Compilation failure of `rustc-serialize` breaks CI

Compilation of `em-app` fails with:

```
em-app$ cargo +nightly-2024-01-05 build --target x86_64-fortanix-unknown-sgx
...
error[E0310]: the parameter type `T` may not live long enough
    --> /home/raoul/.cargo/git/checkouts/rustc-serialize-43e3299849baf4f9/9800ee0/src/serialize.rs:1155:5
     |
1155 |     fn decode<D: Decoder>(d: &mut D) -> Result<Cow<'static, T>, D::Error> {
     |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     |     |
     |     the parameter type `T` must be valid for the static lifetime...
     |     ...so that the type `T` will meet its required lifetime bounds...
     |
note: ...that is required by this bound
    --> /rustc/f688dd684faca5b31b156fac2c6e0ae81fc9bc90/library/alloc/src/borrow.rs:180:30
help: consider adding an explicit lifetime bound
     |
1151 | impl<'a, T: ?Sized + 'static> Decodable for Cow<'a, T>
     |                    +++++++++

For more information about this error, try `rustc --explain E0310`.
error: could not compile `rustc-serialize` (lib) due to 1 previous error
```
Earlier nightlies compiler do not result in this failure. A patch of `rustc-serialize` is required.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -40,4 +40,4 @@ mbedtls = { git = "https://github.com/fortanix/rust-mbedtls", branch = "master" 
 nix   = { git = "https://github.com/fortanix/nix.git", branch = "raoul/fortanixvme_r0.20.2" }
 serde = { git = "https://github.com/fortanix/serde.git", branch = "master" }
 vsock = { git = "https://github.com/fortanix/vsock-rs.git", branch = "fortanixvme" }
-rustc-serialize = { git = "https://github.com/jethrogb/rustc-serialize.git", branch = "portability" }
+rustc-serialize = { git = "https://github.com/fortanix/rustc-serialize.git", branch = "portability" }

